.editorconfig

```diff
@@ -0,0 +1,5 @@
+root = true
```

Comment:
Actually I did... .editorconfig is a standard configuration file that sets the indentation properties for files.  It's recognized by eclipse and intellij.  i can remove it, if you want... but it got added because my default spacing for tabs is not 2 and I got tired of having to keep reformatting stuff :)